src/main/java/uk/gov/companieshouse/overseasentitiesapi/service/update/PopulateUpdateSubmission.java

```diff
@@ -0,0 +1,96 @@
+package uk.gov.companieshouse.overseasentitiesapi.service.update;
+
+import uk.gov.companieshouse.overseasentitiesapi.model.dto.DueDiligenceDto;
+import uk.gov.companieshouse.overseasentitiesapi.model.dto.OverseasEntitySubmissionDto;
+import uk.gov.companieshouse.overseasentitiesapi.model.updatesubmission.DueDiligence;
+import uk.gov.companieshouse.overseasentitiesapi.model.updatesubmission.FilingForDate;
+import uk.gov.companieshouse.overseasentitiesapi.model.updatesubmission.Presenter;
+import uk.gov.companieshouse.overseasentitiesapi.model.updatesubmission.UpdateSubmission;
+
+/**
+ * This is a helper class for FilingsService when the filing is an update. The purpose of this class
+ * is to populate change submission values into UpdateSubmission.java with the ultimate gaol to
+ * produce a json output of change submissions. It uses OESubmissionDTO and UpdateDTO values to
+ * populate JSON template.
+ */
+public class PopulateUpdateSubmission {
+  private OverseasEntitySubmissionDto overseasEntitySubmissionDto;
+  private UpdateSubmission updateSubmission;
+
+  public PopulateUpdateSubmission(
+      OverseasEntitySubmissionDto overseasEntitySubmissionDto, UpdateSubmission updateSubmission) {
+    this.overseasEntitySubmissionDto = overseasEntitySubmissionDto;
+    this.updateSubmission = updateSubmission;
+  }
+
+  /* method populates values into UpdateSubmission for JSON output */
+  public UpdateSubmission populate() {
+    // updateSubmission.type(); Type is already set in UserSubmission, has a default value of
+    // "OE02";
+    updateSubmission.setUserSubmission(this.overseasEntitySubmissionDto);
+    populateDueDiligence(this.overseasEntitySubmissionDto, updateSubmission);
+    populatePresenter(this.overseasEntitySubmissionDto, this.updateSubmission);
+    updateSubmission.setBeneficialOwnerStatement(
+        this.overseasEntitySubmissionDto.getBeneficialOwnersStatement().getValue());
+
+    updateSubmission.setAnyBOsOrMOsAddedOrCeased(
+        this.overseasEntitySubmissionDto.getUpdate().isRegistrableBeneficialOwner());
+    // AC contact details is covered in --> overseasEntitySubmissionDto.getPresenter().getEmail();
+    // TBD: NoChangesInFilingPeriodStatement will be done in
+    // https://companieshouse.atlassian.net/browse/UAR-461//
+    // TBD updateSubmission.setNoChangesInFilingPeriodStatement(); //
+
+    return updateSubmission;
+  }
+
+  public void populateDueDiligence(
+      OverseasEntitySubmissionDto overseasEntitySubmissionDto, UpdateSubmission updateSubmission) {
+    DueDiligenceDto dueDiligenceDto =
```

Comment:
There are two types of due dillegence in the overseasEntitySubmissionDto. Overseas and plain dd. Only one will be non null and that one should be used to populate the DD in the updateSubmission. The fields to add will vary slightly but I hope that is covered in the ODP.